applications/acdc/src/acdc_agent_fsm.erl

```diff
@@ -750,12 +750,10 @@ ringing({'channel_hungup', MemberCallId, _Cause}, #state{agent_listener=AgentLis
                                                          ,account_id=AccountId
                                                          ,agent_id=AgentId
                                                          ,member_call_id=MemberCallId
-                                                         ,member_call_queue_id=QueueId
                                                         }=State) ->
     lager:debug("caller's channel (~s) has gone down, stop agent's call: ~s", [MemberCallId, _Cause]),
     acdc_agent_listener:channel_hungup(AgentListener, MemberCallId),
 
-    acdc_stats:call_abandoned(AccountId, QueueId, MemberCallId, ?ABANDON_HANGUP),
```

Comment:
Perhaps instead of removing these (as others may want these data points), you could introduce calls to `whapps_config:get_is_true(?CONFIG_CAT, <<"track_member_timeout">>, 'true')` to preserve backwards compatibility. For each `ABANDON_*` reason, system admins can select which hangup causes they'd prefer to be tracked.